README_ja.md

```diff
@@ -0,0 +1,103 @@
+	</a>
+	<br>
+</div>
+
+[English](https://github.com/opendilab/LLMRiddles/blob/main/README.md) | [简体中文](https://github.com/opendilab/LLMRiddles/blob/main/README_zh.md) | 日本語
+
+## :thinking: これは何ですか？
+LLM Riddles へようこそ！これは言語モデルとの知恵と勇気のゲームです。ゲームでは、要件を満たす答えを得るために、言語モデルと相互作用する質問を作成する必要があります。このプロセスでは、頭を使い、思いつく限りの方法を駆使して、モデルに答えが求める結果を出力させることができます。
+
+## :space_invader: 遊び方
+私たちは、プレイヤーが直接アクセスして試せるオンライン版を提供しています。
+- [Hugging Face][ChatGPT + 英語(キーが必要)](https://huggingface.co/spaces/OpenDILabCommunity/LLMRiddlesChatGPTEN)
+- [Hugging Face][ChatGPT + 中国語(キーが必要)](https://huggingface.co/spaces/OpenDILabCommunity/LLMRiddlesChatGPTCN)
+- [Hugging Face][ChatGLM + 英語(キー付き)](https://huggingface.co/spaces/OpenDILabCommunity/LLMRiddlesChatGLMEN)
+- [Hugging Face][ChatGLM + 中国語(キー付き)](https://huggingface.co/spaces/OpenDILabCommunity/LLMRiddlesChatGLMCN)
+- [OpenXLab][ChatGPT + 中国語(キーが必要)](https://openxlab.org.cn/apps/detail/OpenDILab/LLMRiddlesChatGPTCN)
+- [OpenXLab][ChatGPT + 英語(キーが必要)](https://openxlab.org.cn/apps/detail/OpenDILab/LLMRiddlesChatGPTEN)
+- [OpenXLab][ChatGLM + 中国語(キー付き)](https://openxlab.org.cn/apps/detail/OpenDILab/LLMRiddlesChatGLMCN)
+- [OpenXLab][ChatGLM + 英語(キー付き)](https://openxlab.org.cn/apps/detail/OpenDILab/LLMRiddlesChatGLMEN)
+- [Private Server][ChatGPT + 中国語(キー付き)](http://llmriddles.opendilab.net/)
+
+ヒントや解決策についての技術ブログも提供しています: [リンク](https://zhuanlan.zhihu.com/p/667801731)
+
+ローカルへのデプロイは以下の方法で行うことができます:
+## インストール
+### ChatGPT / ChatGLM API を使用する
+```shell
+pip3 install -r requirements.txt
+```
+### ローカル推論用に Mistral-7B-Instruct-v0.1 をデプロイする
+```shell
+pip3 install -r requirements-dev.txt
+```
+## 立ち上げ
+### ChatGPT + 中国語
+```shell
+QUESTION_LANG=cn QUESTION_LLM='chatgpt' QUESTION_LLM_KEY=<your API key> python3 -u app.py
+```
+### ChatGPT + 英語
+```shell
+QUESTION_LANG=en QUESTION_LLM='chatgpt' QUESTION_LLM_KEY=<your API key> python3 -u app.py
+```
+### ChatGLM + 中国語
+```shell
+QUESTION_LANG=cn QUESTION_LLM='chatglm' QUESTION_LLM_KEY=<your API key> python3 -u app.py
+```
+### ChatGLM + 英語
+```shell
+QUESTION_LANG=en QUESTION_LLM='chatglm' QUESTION_LLM_KEY=<your API key> python3 -u app.py
+```
+### Mistral-7B-Instruct-v0.1 + 英語
+```shell
+QUESTION_LANG=en QUESTION_LLM='mistral-7b' python3 -u app.py
+```
+## :technologist: なぜこれをするのか
+
+私たちの目標は、このゲームを通じて、プロンプトエンジニアリングと自然言語処理の魅力的な側面を参加者に深く理解してもらうことです。このプロセスでは、プロンプトを巧みに構成する方法や、人工知能システムから驚くような回答を引き出すためのプロンプトの使い方を選手たちに示すと同時に、ディープラーニングや自然言語処理技術の驚異的な力をより深く理解できるようにする。
+
+## :raising_hand: カスタムレベルの提出方法
+面白い質問やアイデアがあれば、プレイヤーは自分のアイデアを提出することができます。あなたは[プルリクエストの開始](https://github.com/opendilab/LLMRiddles/compare)を行い、私たちに送信することができます。承認後、私たちはそれをレベルに含めます。
+質問形式には以下の点を含めること:
+- プルリクエストタイトル, 例: feature(username): Chapter X-Level Design
+- 記載したい ID
+- 対応する章の問題ファイルを修正する
+- \__init__.py の修正
+
+完全な例については、以下を参照のこと: [自分のレベルデザインを提出する](https://github.com/opendilab/LLMRiddles/pull/6)
+
+## :writing_hand: ロードマップ
```

Comment:
Thanks for your contribution, I have a question about this, can this sentence in Japanese also convey the same meaning of future development plans?

Reply:
@TuTuHuss Yes!